File: auto-docs/executables/r/error_bar_asymmetric_constant.r
# Learn about API authentication here: plot.ly/r/getting-started
# Find your api_key here: plot.ly/settings/api

library(plotly)
py <- plotly(username='TestBot', key='r1neazxo9w')


data <- list(
  list(
    x = c(1, 2, 3, 4), 
    y = c(2, 1, 3, 4), 
    error_y = list(
      type = "percent", 
      symmetric = FALSE, 
      value = 15, 
      valueminus = 25
    ), 
    type = "scatter"
  )
)
response <- py$plotly(data, kwargs=list(filename="error-bar-asymmetric-constant", fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/julia/error_bar_asymmetric_constant.jl
using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

data = [
  [
    "x" => [1, 2, 3, 4], 
    "y" => [2, 1, 3, 4], 
    "error_y" => [
      "type" => "percent", 
      "symmetric" => false, 
      "value" => 15, 
      "valueminus" => 25
    ], 
    "type" => "scatter"
  ]
]
response = Plotly.plot(data, ["filename" => "error-bar-asymmetric-constant", "fileopt" => "overwrite"])
plot_url = response["url"]
